main/util/src/mill/util/Jvm.scala

```diff
@@ -454,6 +630,23 @@ object Jvm extends CoursierSupport {
     )(new Ctx.Home { override def home = os.home })
   }
 
+  def callClassLoader[T](
+      classPath: Iterable[os.Path],
+      sharedPrefixes: Seq[String] = Seq.empty,
+      parent: ClassLoader = null
+  )(f: ClassLoader => T)(implicit ctx: mill.api.Ctx.Home): T = {
+    val oldClassloader = Thread.currentThread().getContextClassLoader
+    val newClassloader = spawnClassloader(classPath, sharedPrefixes, parent)
```

Comment:
Let's define a `Jvm.createClassloader` and deprecate `mill.api.ClassLoader.create`. This will help make it discoverable, since it'll be defined in the same place as all the other  classloader/subprocess methods in `Jvm.*`.

Reply:
Ok, np.  
Should I leave deprecation annotation "since 0.12.7" as is now? `@deprecated("Use callClassLoader", "Mill 0.12.7")`  
Or put `0.13.0-M1` or?

Reply:
@sake92 let's say deprecated `since 0.12.8`, with the goal of backporting this PR from `main` back to `0.12.x` so that it can go out in the next `0.12.8` release